### DISCUSSION TITLE
Presto DB

### DISCUSSION THREAD

**Discussion by AmiParadkar** [prisma/prisma#9091] [Q&A] [answered]
Hi 

We have data in PrestoDB that we want to extract using GraphQL, is there any way to connect to PrestoDB from Prisma?

**Comment by janpio** [comment, marked as answer]
I don't think so, and according to our feature requests issues this has never been asked before. I suggest you also create a feature request issue, so we can see if other people are interested in this as well.

**Reply by ryands17** [reply]
@AmiParadkar 👋 

It would be great if you could create one [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) 🙂

**Reply by mingfang** [reply]
https://github.com/prisma/prisma/issues/15080

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
